mm2src/coins/utxo/utxo_standard.rs

```diff
@@ -313,6 +315,23 @@ impl SwapOps for UtxoStandardCoin {
     fn extract_secret(&self, secret_hash: &[u8], spend_tx: &[u8]) -> Result<Vec<u8>, String> {
         utxo_common::extract_secret(secret_hash, spend_tx)
     }
+
+    fn can_refund_htlc(&self, locktime: u64) -> Box<dyn Future<Item = CanRefundHtlc, Error = String> + Send + '_> {
```

Comment:
Is this logic valid for Qtum coin? I guess the `can_refund_htlc` function can be moved to `utxo_common` module.

Reply:
Yeah, I've rechecked and it looks like QTUM also activated the BIP113. I'm going to move the code now.

Reply:
Done, please review again :slightly_smiling_face: